src/main/java/io/xdb/core/client/ServerApiRetryConfig.java

```diff
@@ -0,0 +1,22 @@
+package io.xdb.core.client;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Builder
+@Getter
+public class ServerApiRetryConfig {
+
+    private long initialIntervalMills;
+    private long maximumIntervalMills;
+    private int maximumAttempts;
+
+    public static ServerApiRetryConfig getDefault() {
+        return ServerApiRetryConfig
+            .builder()
```

Comment:
Nice, looks like lombok is better/friendly than immutable 👍 I had to manually add those builder helpers myself when using immutable